genesyscloud/process_automation_trigger/resource_genesyscloud_processautomation_trigger.go

```diff
@@ -39,6 +51,14 @@ var (
 				Type:        schema.TypeString,
 				Required:    true,
 			},
+			"workflow_target_settings": {
+				Description: "Special settings related to workflow target invocation",
```

Comment:
I usually copy and paste the definition in the swagger file to be safe for this field it is: "Optional config for the target. Until the feature gets enabled will always operate in TopLevelPrimitives mode."

Reply:
(If you decide to change them just be sure to run `go generate` again)